README.md

```diff
@@ -1,75 +1,118 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/ae60f2a4-488e-4771-b24a-c26badc5f45d/deploy-status)](https://app.netlify.com/sites/kongdocs/deploys)
+[![](https://img.shields.io/github/license/kong/docs.konghq.com)](https://github.com/Kong/docs.konghq.com/blob/main/LICENSE)
+[![](https://img.shields.io/github/contributors/kong/docs.konghq.com)]()
 
 # KONG's Documentation Website
 
-This repository provides the source code and content for [Kong](https://github.com/Kong/kong)'s documentation website. It is a [Jekyll](https://jekyllrb.com/) website deployed with Netlify.
+This repository holds source code for [Kong](https://github.com/Kong/kong)'s documentation website. It's built using [Jekyll](https://jekyllrb.com/) and deployed with [Netlify](https://www.netlify.com/). 
 
-Not sure where to start? Head on over to the `issues` tab to and look for the `good first issue` label. These are issues Kong has identified as beginner friendly. Many of these can be addressed with the GitHub UI and do not require pulling the repository and building locally.
+Here are some things to know before you get started:
 
-First-time contributors, check out our [contributing guide on the website](https://docs.konghq.com/contributing/), and the linked resources there.
+* **We're beginner-friendly**. Whether you're a Technical Writer getting into code-as-docs or an engineer practicing your documentation skills, we highly encourage your involvement in our project. If you'd like to contribute and don't have something in mind already, head on over to [Issues](https://github.com/Kong/docs.konghq.com/issues). We've added `good first issue` labels on beginner-friendly issues.
 
-## Building locally
+* **We need more help in some areas**. We'd especially love some help with [plugin](https://github.com/Kong/docs.konghq.com/tree/main/app/_hub) documentation. 
 
-If you're making more than a small typo or grammar change, we ask that you pull down the repository and build locally.
+* **Some of our docs are auto-generated**. 
+    * [Admin API](https://docs.konghq.com/gateway-oss/)
+    * [Configuration reference](https://docs.konghq.com/gateway-oss/latest/configuration/)
+    * [CLI reference](https://docs.konghq.com/gateway-oss/latest/cli/)
+    * [Upgrade guide](https://docs.konghq.com/gateway-osslatest/upgrading/) 
 
-### Develop locally with Docker
+All PRs for Admin API should be open over at the [Kong/kong](https://github.com/Kong/kong) repository.
```

Comment:
```suggestion
All PRs for these docs should be opened over at the [Kong/kong](https://github.com/Kong/kong) repository.
```
(since there's more than just the Admin API)